packages/core/src/codewhispererChat/commands/registerCommands.ts

```diff
@@ -37,73 +36,13 @@ export const focusAmazonQPanelKeybinding = Commands.declare('_aws.amazonq.focusC
-    Commands.register('aws.amazonq.explainCode', async (data) => {
-        return focusAmazonQPanel.execute(placeholder, 'amazonq.explainCode').then(() => {
-            controllerPublishers.processContextMenuCommand.publish({
-                type: 'aws.amazonq.explainCode',
-                triggerType: getCommandTriggerType(data),
-            })
-        })
-    })
-    Commands.register('aws.amazonq.refactorCode', async (data) => {
-        return focusAmazonQPanel.execute(placeholder, 'amazonq.refactorCode').then(() => {
-            controllerPublishers.processContextMenuCommand.publish({
-                type: 'aws.amazonq.refactorCode',
-                triggerType: getCommandTriggerType(data),
-            })
-        })
-    })
-    Commands.register('aws.amazonq.fixCode', async (data) => {
-        return focusAmazonQPanel.execute(placeholder, 'amazonq.fixCode').then(() => {
-            controllerPublishers.processContextMenuCommand.publish({
-                type: 'aws.amazonq.fixCode',
-                triggerType: getCommandTriggerType(data),
-            })
-        })
-    })
-    Commands.register('aws.amazonq.optimizeCode', async (data) => {
-        return focusAmazonQPanel.execute(placeholder, 'amazonq.optimizeCode').then(() => {
-            controllerPublishers.processContextMenuCommand.publish({
-                type: 'aws.amazonq.optimizeCode',
-                triggerType: getCommandTriggerType(data),
-            })
-        })
-    })
-    Commands.register('aws.amazonq.sendToPrompt', async (data) => {
-        return focusAmazonQPanel.execute(placeholder, 'amazonq.sendToPrompt').then(() => {
-            controllerPublishers.processContextMenuCommand.publish({
-                type: 'aws.amazonq.sendToPrompt',
-                triggerType: getCommandTriggerType(data),
-            })
-        })
-    })
-    Commands.register('aws.amazonq.explainIssue', async (issue) => {
-        return focusAmazonQPanel.execute(placeholder, 'amazonq.explainIssue').then(() => {
-            controllerPublishers.processContextMenuCommand.publish({
-                type: 'aws.amazonq.explainIssue',
-                triggerType: 'click',
-                issue,
-            })
-        })
-    })
-    Commands.register('aws.amazonq.generateUnitTests', async (data) => {
-        return focusAmazonQPanel.execute(placeholder, 'amazonq.generateUnitTests').then(() => {
-            controllerPublishers.processContextMenuCommand.publish({
-                type: 'aws.amazonq.generateUnitTests',
-                triggerType: getCommandTriggerType(data),
-            })
-        })
-    })
-    Commands.register('aws.amazonq.updateContextCommandItems', () => {
-        controllerPublishers.processContextCommandUpdateMessage.publish()
-    })
+export function registerCommands() {
+    /**
+     * make these no-ops, since theres still callers that need to be deprecated
```

Comment:
are these commands still exposed to the user? Should we disable them until they are hooked up?

Reply:
They're only still enabled because other parts of the extension (like security scan) call them. In a future PR i'm going to fully deprecate them here and implement them in commands.ts but for now this just avoids spamming the logs with command not found